api/threem.py

```diff
@@ -603,15 +604,38 @@ class ThreeMEventMonitor(Monitor):
     associated with it until the ThreeMCirculationMonitor runs.
     """
 
+    TWO_YEARS_AGO = datetime.timedelta(365*2)
+    name = "3M Event Monitor"
+
     def __init__(self, _db, default_start_time=None,
                  account_id=None, library_id=None, account_key=None):
+        if not default_start_time:
+            default_start_time = self.create_default_start_time(sys.argv[1:])
```

Comment:
For testability I like to keep the sys.argv stuff in the python script that's kept in bin/, not in these classes (the main exception is IdentifierInputScript, where the sys.argv management is in its own method). This isn't a big deal but see if you can move this to 3m_monitor.